src/panel.js

```diff
@@ -14,6 +14,10 @@ import { getAllLidiaAnnotations } from "./relations.js";
  * as a whole - this panel is shared between Zotero tabs.
  */
 export class LidiaPanel {
+    tab;
+    tabPanel;
+    annotationEvents = [];
```

Comment:
These three lines are not valid in ES2015, so I added the transpiling option. We may also delete these lines but I suggest keeping the transpiling enabled to avoid confusion.